### PR TITLE
add test case for get transaction of types

### PR DESCRIPTION
**Pull request by darkdarkdragon**
suspendedPaymentExecution, suspendedPaymentCancellation, suspendedPaymentCreation

### CODE REVIEW

File: test/fixtures/requests/prepare-suspended-payment-execution.json [hunk 1]
```diff
@@ -3,5 +3,5 @@
   "suspensionSequence": 1234,
   "method": 1,
   "digest": "8F434346648F6B96DF89DDA901C5176B10A6D83961DD3C1AC88B59B2DC327AA4",
-  "proof": "whatever"
+  "proof": "whateverthis bemusthave 32 chars"
```
Comment (clark800) [inline test/fixtures/requests/prepare-suspended-payment-execution.json]: Does the digest match this proof? Could you make it a little cleaner like "this must have 32 characters...."?

**Comment by clark800** [comment]
Is there any way to add integration tests for this functionality?

**Comment by clark800** [comment]
Does this depend on the changes to `ripple-binary-codec`?

**Comment by darkdarkdragon** [comment]
yes, it depends on `ripple-binary-codec`
Integration tests can be added using docker images of `rippled` develop branch running in standalone mode

**Comment by clark800** [comment]
Do we have to update the `ripple-binary-codec` dependency version then? I released your changes as version 0.0.7.

There aren't any test servers running rippled with suspended payments enabled?

File: test/fixtures/requests/prepare-suspended-payment-execution.json (hunk 1 above)
Comment (darkdarkdragon) [inline test/fixtures/requests/prepare-suspended-payment-execution.json]: I've changed proof and made digest to correspond to it

**Comment by darkdarkdragon** [comment]
I've updated `ripple-binary-codec` dependency version.

> There aren't any test servers running rippled with suspended payments enabled?
I don't know. I've just compiled it myself.

**Comment by clark800** [comment]
You would also have to update the shrinkwrap file using the `npm-shrinkwrap` command (not `npm shrinkwrap`). But first, you should add a test case that fails with the `ripple-binary-codec` v0.0.6 so we know that we are testing it properly.

**Comment by darkdarkdragon** [comment]
OK

**Comment by clark800** [comment]
LGTM

**Pull request merged**